src/index.html

```diff
@@ -1,13 +1,284 @@
+            </li>
+
+            <li class="menu__item">
+              <a href="#gallary" class="menu__link">Gallary</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#testimonial" class="menu__link">Testimonial</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact us</a>
+            </li>
+
+          </ul>
+
+          <a
+            href="+1 234 555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 555-55-55
+          </a>
+
+          <a
+            href="+1 234 555-55-55"
+            class="menu__call menu__call--action"
+          >
+            call to order
+          </a>
+
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Miami best
+                <br>
+                Real estate
+                <a href="#" class="about__title-link">
+                  here
+                </a>
+              </h2>
+
+            </div>
+
+            <div class="grid__item grid__item--7-12">
+              <a href="#" class="about__subtitle">
+                Over 50 years experience in love.
+              </a>
+
+              <p class="about__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+              </p>
+
+              <p class="about__paragraph">
+                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.
+              </p>
+
+              <p class="about__paragraph about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p>
+
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section gallary" id="gallary">
+        <div class="container">
+          <h2 class="page__section-title">
+            Gallery
+          </h2>
+
+          <div class="gallary__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-8">
+                  <div class="place__photo-container">
+                    <img
+                    src="./images/photos/1.jpg"
+                    alt="Place 1"
+                    class="place__photo"
+                    >
```

Comment:
pay attention to the indentations, fix in all places
```suggestion
                    <img
                      src="./images/photos/1.jpg"
                      alt="Place 1"
                      class="place__photo"
                    >
```